packages/scraper/src/actions/CalenderAction.ts

```diff
@@ -0,0 +1,29 @@
+import { CalenderAction } from './../interfaces/ActionInterfaces/CalenderInterface';
+import { RootState } from './../reducers/index';
```

Comment:
```suggestion
import { RootState } from './../reducers';
```
아마 이렇게 하셔도 될거에용(아마?)
기본적으로 index를 보고있어서

Reply:
좋습니다~